### ISSUE TITLE
Unable to initialize the Plot plugin. ReactNative

### ISSUE THREAD

**Issue by ElenaMironovaA** [Plotprojects/plotprojects-react-native-module#7]
Hi!

I have tried to use PlotProjects but faced with a problem:

```
[85735:4828593] ERROR|PlotInterface| Unable to initialize the Plot plugin. No public token set.
```
<img width="860" alt="Screenshot 2022-11-11 at 16 42 48" src="https://user-images.githubusercontent.com/108752263/201388823-ad0a3cdd-6061-49f4-931e-b077afa43ea7.png">

<img width="762" alt="image" src="https://user-images.githubusercontent.com/108752263/201389625-36170f73-7e63-42f4-a616-099719d67a7f.png">

<img width="1124" alt="Screenshot 2022-11-11 at 16 47 38" src="https://user-images.githubusercontent.com/108752263/201389222-ab40e7bc-0410-455d-8a62-f3342f33288a.png">

Also I run RN app example, and there is the same issue.
Could you help me resolve this problem?

**Comment by YahyaAlfayad** [comment]
Hello Elena,

Have you tried running the example app here https://github.com/Plotprojects/plot-react-native-module-example after modifying the existing `plotconfig.json` file inside `Supporting Files` directory?

**Comment by ElenaMironovaA** [comment]
Yep, there is the same issue

**Comment by YahyaAlfayad** [comment]
I've tested with the example app and everything seems to be fine. I was able to receive notifications.

Are you able to receive notifications?

Can you please send me your log file so I can look into this in more details?

**Comment by ElenaMironovaA** [comment]
Yep, my app receives a notification as well, but I think this log(error) can be a little bit concern developers. Probably do you able to fix it?
